lib/rules/max-lines.js

```diff
@@ -127,14 +140,35 @@ module.exports = {
                 if (skipComments) {
                     const comments = sourceCode.getAllComments();
 
-                    const commentLines = lodash.flatten(comments.map(comment => getLinesWithoutCode(comment)));
+                    const commentLines = lodash.flatten(
+                        comments.map(comment => getLinesWithoutCode(comment))
+                    );
 
-                    lines = lines.filter(l => !lodash.includes(commentLines, l.lineNumber));
+                    lines = lines.filter(
+                        l => !lodash.includes(commentLines, l.lineNumber)
+                    );
                 }
 
+                lines.forEach((line, i) => {
+                    if (max + 1 === i + 1) {
+                        startLine = line.lineNumber;
+                    }
+                });
```

Comment:
Could this be replaced with just `const startLine = lines[max].lineNumber;` inside `if (lines.length > max`